client/dashboard/app/router/me.tsx

```diff
@@ -773,6 +774,53 @@ export const appsRoute = createRoute( {
 	)
 );
 
+export const mcpRoute = createRoute( {
+	head: () => ( {
+		meta: [
+			{
+				title: __( 'MCP Account Settings' ),
+			},
+		],
+	} ),
+	getParentRoute: () => meRoute,
+	path: 'mcp',
+} );
+
+export const mcpIndexRoute = createRoute( {
+	getParentRoute: () => mcpRoute,
+	path: '/',
+	loader: async () => {
+		await queryClient.ensureQueryData( userSettingsQuery() );
```

Comment:
nit: We can move it to `mcpRoute` so we don't need it on both `mcpIndexRoute` and `mcpSetupRoute`